archr/analyzers/__init__.py

```diff
@@ -2,6 +2,7 @@
 import time
 import logging
 from contextlib import contextmanager
+from ..targets.actions import OpenChannelAction, SendAction, CloseChannelAction
```

Comment:
Local imports can be their own section, also is this possible as an absolute import?

Reply:
sure they can have their own section. But why absolute import?

Reply:
Absolute imports are just easier to read in most cases, and intra-package relative imports are discouraged in various peps. Don't bother if it creates problems, but if either works then absolute is preferable.

Reply:
OK. absolute import it is then. thanks for the info